backend/aws/lambda/app.py

```diff
@@ -5,34 +5,65 @@
 import base64
 from dataclasses import dataclass
 import time
+import boto3
+from botocore.exceptions import ClientError
 
 # TODO: These should be env variables or imported from something better
 versions = {'model': 0.1, 'lambda': 0.1, 'api': 0.1, 'extension': 0.1}
-
-rds_host  = "deepcite.ckbyp3nhsmiu.us-east-2.rds.amazonaws.com"
+secret_name = "rds_deepcite_sample"
+region_name = "us-east-2"
 db_name = 'postgres'
-db_password = 'deepcite'
-db_username = 'postgres'
 
-print('Loading function')
+# Create a Secrets Manager client
+session = boto3.session.Session()
+secret_client = session.client(
+    service_name='secretsmanager',
+    region_name=region_name
+)
 
 # private ip address of ec2
 url = 'http://172.31.35.42:8000/api/v1/deep_cite'
 
-try:
-    conn = psycopg2.connect(host=rds_host, user=db_username, password=db_password, database=db_name, port='5432')
-except psycopg2.OperationalError as e:
-    print("ERROR: Unexpected error: Could not connect to database instance.")
-    print(e)
-    sys.exit()
-print("SUCCESS: Connection to RDS instance succeeded")
+
+def get_secret():
+    print('grabbing secret')
+    try:
+        get_secret_value_response = secret_client.get_secret_value(
+            SecretId=secret_name
+        )
+    except ClientError as e:
+        if e.response['Error']['Code'] == 'DecryptionFailureException':
+            # Secrets Manager can't decrypt the protected secret text using the provided KMS key.
```

Comment:
Should these comments also be printed out with the error?

Reply:
You are absolutely right, they should be

Reply:
I'm just going to remove them for now because they should be something you can just google.